client/src/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,17 @@
+import Container from 'react-bootstrap/Container';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import { useState } from "react";
+import Button from "react-bootstrap/Button";
+import {useNavigate} from "react-router-dom";
+
+export default function MainContainer() {
+
+    let navigate = useNavigate();
+    const LoginRoute = () =>{
```

Comment:
This function isn't necessary.  It isn't called in the component and would route to the same component.